plugin/evm/vm.go

```diff
@@ -660,7 +660,7 @@ func (vm *VM) initializeChain(lastAcceptedHash common.Hash) error {
 		&vm.ethConfig,
 		&EthPushGossiper{vm: vm},
 		vm.chaindb,
-		vm.config.EthBackendSettings(),
+		eth.Settings{MaxBlocksPerRequest: vm.config.MaxBlocksPerRequest},
```

Comment:
why did we need to change this?

Reply:
config package shouldn't import "eth" package (for eth.Settings)
overall config packages for use by clients should import as little as possible.

Reply:
I wasn't even aware that we had such function in there (I thought this was eth.Settings already lol)